theforeman.org/pipelines/vars/candlepin/4.3.groovy

```diff
@@ -3,7 +3,7 @@ def candlepin_distros = ['el8', 'el9']
 def packaging_branch = 'rpm/4.3'
 def pipelines = [
     'candlepin': [
-        'centos8-stream',
-        'centos9-stream'
+        'centos9-stream',
+        'almalinux8'
```

Comment:
It wasn't done before, but I prefer trailing commas so you can easily add other things later.